setup.py

```diff
@@ -31,6 +31,9 @@
             "FITSVarBintable = kerchunk.codecs:VarArrCodec",
             "record_member = kerchunk.codecs.RecordArrayMember",
         ],
+        'console_scripts': [
+            'kerchunk-nc = kerchunk.cli.chunk_nc:cli',
```

Comment:
let's just call it `kerchunk`, and either infer the file type from the URL extension or provide a --format option to select nc, the only one available right now.